### ISSUE TITLE
Introduction

### ISSUE THREAD

**Issue by ns-rse** [ns-rse/git-collaboration#67]

**Comment by Robadob** [comment]
Worth linking to new repo page, when asking user to create a new repo. https://github.com/new

**Comment by Robadob** [comment]
Repository owner 3: (why not `git remote set-url origin <url>` ?)
Repository owner 4: An unforced push should be sufficient if people haven't initialise the repo with other stuff.

**Comment by Robadob** [comment]
Protect the main branch, maybe worth adding a step 4 telling them to submit the form (it's not ajax magic)

**Comment by Robadob** [comment]
Callout at the bottom

> If desired you can between you update the Metadata in pyproject.toml 

Maybe reword, also explain what the benefit of updating pyproject.toml is or link to somewhere that explains it. If explaining it is irrelevant, then so is the callout/.toml.
